cxx-sensors/src/main/java/org/sonar/cxx/sensors/clangsa/CxxClangSASensor.java

```diff
@@ -77,6 +77,8 @@ protected void processReport(final SensorContext context, File report)
 
       NSDictionary rootDict = (NSDictionary) PropertyListParser.parse(f);
 
+      final String clangVersion = ((NSString) rootDict.get("clang_version")).getContent();

```

Comment:
Why not checking for the clang version directly here?

Reply:
I thought about this. Checking version would only work for stock builds of clang. Imagine someone builds custom version of clang with the version string like "rh-3.7.1-internal". For such compiler checking version would not work.